worlds/rogue_legacy/Regions.py

```diff
@@ -1,31 +1,27 @@
 from typing import Dict, List, NamedTuple, Optional
 
 from BaseClasses import MultiWorld, Region, RegionType, Entrance
-from .Items import RLItem
 from .Locations import RLLocation, location_table, get_locations_by_category
 
 
 class RLRegionData(NamedTuple):
     locations: Optional[List[str]]
-    exits: Optional[List[str]]
+    region_exits: Optional[List[str]]
 
 
 def create_regions(multiworld: MultiWorld, player: int):
```

Comment:
The arg here could just be World instead of using two separate args. Alternatively you have the data root level in the module and do everything in this function within the world itself.

Reply:
I disagree. If I only send in `RLWorld`, then I have two options:

- I need to pull the `MultiWorld` and player id off into separate variables (adding 2 lines for no additional clarity).
- I reference the data from RLWorld inline, and my lines are longer.

I prefer just separating them once when calling the function and then never again.